framework/src/transfers/MultiAppNearestNodeTransfer.C

```diff
@@ -893,3 +862,35 @@ MultiAppNearestNodeTransfer::getLocalEntities(
     }
   }
 }
+
+std::vector<Node *>
+MultiAppNearestNodeTransfer::getTargetLocalNodes(const unsigned int to_problem_id)
+{
+  std::vector<Node *> target_local_nodes;
```

Comment:
Make `std::vector<Node *> target_local_nodes;` as a private member of the class. Change its name to `_target_local_nodes`.  Do _target_local_nodes.clear() at the beginning of `MultiAppNearestNodeTransfer::getTargetLocalNodes`.

Reply:
Ah, gotcha - thanks for explaining! I ended up just changing the function to `setTargetLocalNodes` which sets the member variable - just seemed a little simpler instead of returning something, but can easily change it if you prefer.

Reply:
The old way seems more natural for me.

Reply:
Should be ready now! Thanks